myocamlbuild.ml

```diff
@@ -941,7 +941,7 @@ dispatch
                         "gettext" 
                         (BaseEnvLight.load
                            ~allow_empty:true
-                           ~filename:(Lazy.force MyOCamlbuildBase.env_filename)
+                           ~filename:(MyOCamlbuildBase.env_filename)
```

Comment:
On Sun, 4 Oct 2015 11:28:35 -0700, gerdstolpmann wrote:
> 
> In myocamlbuild.ml:
> 
> > @@ -941,7 +941,7 @@ dispatch
> >                          "gettext"
> >                          (BaseEnvLight.load
> >                             ~allow_empty:true
> > -                           ~filename:(Lazy.force MyOCamlbuildBase.env_filename)
> > +                           ~filename:(MyOCamlbuildBase.env_filename)
> 
> Yes, and I guess this explains it. I was using oasis-0.4.5 as released.
> 
> So, what's the right bootstrapping procedure then?

After you succeed to compile oasis, you should regenerate setup.ml (and the other files) with the generated executable.  Commit that.